src/components/navbar/logo.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
```

Comment:
Move this into the index.js file you will create at components/navbar and export it